hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenCreateSpecs.java

```diff
@@ -200,6 +200,8 @@ private HapiSpec validateNewTokenAssociations() {
                                         selfDenominatedFixedCollector,
                                         otherSelfDenominatedFixedCollector)
                                 .via(creationTxn),
+                        // TODO: Here it throws that we expect INVALID_TREASURY_ACCOUNT_FOR_TOKEN but it was OK.
+                        // When I change it to expect OK it now returns PLATFORM_NOT_ACTIVE
```

Comment:
Created a story for this: https://github.com/hashgraph/hedera-services/issues/8757

Reply:
`PLATFORM_NOT_ACTIVE` could be because of some error or because state locally is not cleaned. This might not be the root cause for this test failure.